387. First Unique Character in a String.java

```diff
@@ -0,0 +1,20 @@
+public class Solution {
```

Comment:
This algorithm is correct, but there is a more elegant solution involving a `set`.